chemprop/models/model.py

```diff
@@ -173,6 +173,10 @@ def _evaluate_batch(self, batch) -> list[Tensor]:
         targets = targets.nan_to_num(nan=0.0)
         preds = self(bmg, V_d, X_f)
 
+        if isinstance(self.predictor, MveFFN):
+            mean, _ = torch.chunk(preds, self.predictor.n_targets, 1)
+            preds = mean
```

Comment:
If I understand correctly, you add this type check here because an `MveFFN` predicts two targets, `mean` and `var` and when computing a metric we only need the mean. I see `EvidentialFFN`s also predict multiple things (mean, v, alpha, beta). I assume we would also need to include a type check for `EvidentialFFN`s. 

We could do that, but it may be better to require `metric` to take the whole raw `preds` and internally split it into the components. That is what the loss functions do (see `MVELoss` and `EvidentialLoss`). I notice many of the metric classes in `metrics.py` just inherit from their corresponding loss and then change nothing, so that they are really just an alias for the loss class. I think we should do that for MVELoss and EvidentialLoss and include them as the `_default_metric` for MveFFN and EvidentialFFN. Then we don't need the type checking here as `mertic` should take care of spliting the predictions.

Reply:
That make sense! I split this part of changes relevant to `MveFFN` into https://github.com/chemprop/chemprop/pull/664. I will work on them later as they are more in the scope of v2.1.